packages/desktop-client/src/components/budget/util.ts

```diff
@@ -11,19 +11,22 @@ import { type DropPosition } from '../sort';
 
 import { getValidMonthBounds } from './MonthsContext';
 
-export function addToBeBudgetedGroup(groups: CategoryGroupEntity[]) {
+export function addToBeBudgetedGroup(
+  groups: CategoryGroupEntity[],
+  t: (key: string) => string,
```

Comment:
Would it be better if we use `getI18n` here and get the `t` method from there?

Reply:
Yep, would work just fine. Don't you mean `i18next` though? I'm happy to oblige, just tell me what you prefer :) Same goes for the other function you commented on.

Reply:
@joel-jeremy ?